src/Zicht/Bundle/HtmldevBundle/Resources/views/_base.html.twig

```diff
@@ -1,26 +1,24 @@
+{% set version = asset_version('/') -%}
 <!doctype html>
 <html class="s-no-javascript  not-loaded  {% block html_class '' %}" {% block html_attributes '' %}>
     <head>
         <meta charset="UTF-8">
         <meta http-equiv="x-ua-compatible" content="ie=edge">
-        <title>{%- if app.environment != 'production' %}{{ asset_version('/') }} - {% endif %}{% block head_title '' %}</title>
+        <title>{%- if app.environment != 'production' and version is not empty %}{{ version ~ ' - ' }}{% endif %}{% block head_title styleguide.title %}</title>
```

Comment:
We hebben meestal ook nog een `staging` omgeving :man_facepalming:

Reply:
Ja klopt. Daar mag de versie gewoon getoond worden toch? Zo zit het ook in de meeste huidige projecten. Heb alleen een case toegevoegd voor als de versie leeg is.

Reply:
In welke situatie is de versie leeg?  Dat lijkt me namelijk een gevolg van een onjuiste deploy or onjuiste development omgeving config...

Reply:
Of een setup waarin er nog geen deploy heeft plaatsgevonden.